frontend/src/lib/components/LemonModal/LemonModal.scss

```diff
@@ -1,57 +1,100 @@
-.LemonModal {
-    .ant-modal-close-x {
-        position: absolute;
-        top: 0.75rem;
-        right: 0.75em;
-        height: auto;
-        width: auto;
-    }
-    .ant-modal-header,
-    .ant-modal-body,
-    .ant-modal-footer {
-        padding: 1rem;
-    }
-    .ant-modal-header,
-    .ant-modal-footer {
-        display: flex;
-        align-items: center;
-        line-height: 1.5rem;
-        border-color: var(--border);
-    }
+.LemonModal__overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    transition: background-color 250ms ease-out, backdrop-filter 250ms ease-out;
+    z-index: var(--z-modal);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
 
-    .ant-modal-header {
-        border-bottom: none;
+    &.ReactModal__Overlay--after-open {
+        background-color: rgba(0, 0, 0, 0.4);
+        backdrop-filter: blur(5px);
     }
 
-    .ant-modal-title {
-        font-weight: 600;
-        width: 100%;
+    &.ReactModal__Overlay--before-close {
+        background-color: rgba(0, 0, 0, 0);
+        backdrop-filter: blur(0px);
     }
+}
+
+.LemonModal {
+    position: relative;
+    display: inline-block;
+    max-width: 80%;
+    min-width: 200px;
+    width: 480px;
+    margin: 1rem auto;
+    border-radius: var(--radius);
+    background-color: #fff;
+    box-shadow: 0 16px 16px -16px rgba(0, 0, 0, 0.35);
+
+    transition: opacity 250ms ease-out, transform 250ms ease-out;
+    display: flex;
+    flex-direction: col;
+    max-height: 90%;
+
+    // Transition properties
+    will-change: transform;
+    transform: scale(0.85) translateY(25%);
```

Comment:
The blur and scaling effects make for a nice experience, but I'm not so sure about the translate. Previously the translate effect originated where the click happened, and I don't think we need to keep that, but now the modal originates from a completely opposite place, which doesn't _feel_ ideal. I think it'd be nicer without `translate()`. Especially since the modal _disappears_ into the center in this implementation.

Reply:
Fair arguments. The animation isn't mentioned in the Design System so far. I thinkI could make it come from the direction of the button clicked with a _relatively_ small amount of code. Antd's implementation of this is pretty lightweight

Reply:
![2022-08-09 16 56 39](https://user-images.githubusercontent.com/2536520/183683764-56ae1c4a-63d0-46d4-84d7-bfec051d2a6a.gif)

@clarkus what do you think is the best compromise here?